### ISSUE TITLE
New device request: ThinkPad Helix compatible but unsupported

### ISSUE THREAD

**Issue by Agvantibo** [n4ru/1vyrain#85]
Data provided by the script:
Machine: Helix
BIOS: GFET35WW
Version: 1.14
Flashsize: 12288

**Comment by digmorepaka** [comment]
If you have a flasher to recover a brick, you can attempt to run the already existing patches for this generation.

https://github.com/digmorepaka/thinkpad-firmware-patches

**Comment by rzfzr** [comment]
@Agvantibo Have you done the procedure? I have a first gen helix aswell (3rd gen intel)
But I'm on GFET65WW 1.44

**Comment by Xanth0k1d** [comment]
What for though? I dont think you can find a wlan/wwan card with compatible form factor.
WWAN is connected via a ribbon cable, the WLAN has a weird form factor.

**Comment by rzfzr** [comment]
I wanted the performance settings, every single time I boot up, dock/undock, I have to reset my profile on throttlestop, because the default settings makes the computer unusable for me, slow to even open a browser...
Particularly undocked, I believe it drops to ~8w, unlocked it peaks at ~14

**Comment by Xanth0k1d** [comment]
> I wanted the performance settings, every single time I boot up, dock/undock, I have to reset my profile on throttlestop, because the default settings makes the computer unusable for me, slow to even open a browser...
> Particularly undocked, I believe it drops to ~8w, unlocked it peaks at ~14

🤔 The wifi is the big no for me.
Well I mean I opened a thread for x131e, and just nice I have a helix 1st gen around, it's the i7 model. Lemme just figure out where the chip is (if its a single 12mb chip then I dont this is going to work out.)

**Comment by Xanth0k1d** [comment]
> > if its a single 12mb chip then I dont this is going to work out
> 
> Doesn't exist for SPI 25xx series. Thinkpads of this era use a 4M+8M in dual SPI mode. Shows up as "Opaque flash chip" 

Yeah, I recall seeing that while doing an internal flash with flashrom.

**Comment by rzfzr** [comment]
No mic issues on my end

**Comment by digmorepaka** [comment]
Reminder that this is an 1vyrain issues section, not a chatroom.
